main.go

```diff
@@ -335,6 +338,8 @@ func (c *Controller) updateVPAMetrics(vpa *autoscaling.VerticalPodAutoscaler) er
 				labelContainer:      containerName,
 				labelRecommendation: recommendation,
 				labelResource:       resource,
+				labelTargetRefName:  vpa.Spec.TargetRef.Name,
```

Comment:
Should we have pod names? That would help us in the grafana pods dashboard directly.

Reply:
We don't need a pod label for dashboards. We can use the `labelTargetRefName` and then use regex to match the pod name. This will also introduce less cardinality into the metrics.